node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+
+const fs = require('fs');
+
+const arguementLength = process.argv.length;
+
+for (let i = 2; i < arguementLength; i++) {
+  setTimeout(() => {
+    fs.readFile(process.argv[i], 'utf8', (err, data) => {
+      if (err) throw err;
+
+      console.log(data);
+    }, 0);
+  });
```

Comment:
For the extra section, I forgot to ban `setTimeout`. Go ahead and remove this file, add, commit, and push again. Once you get approval for the basic levels, you can try this extra challenge in your free time.